.gitignore

```diff
@@ -1,8 +1,2 @@
-pkg
-bin
-hound.sublime-workspace
-data
 .vagrant
-pub/index.html
-pub/excluded_files.html
-config.json
```

Comment:
I think I'll leave config.json to be safe. I suspect it's going to be in a different directory for most people now. But it's good to ensure that nobody sends one in. I'll also leave the .sublime-workspace file. I've actually moved mine out of the source directory but maybe I'll symlink it back. I'm not sure how other people hack on github projects, I make my Hound directory into a workspace so that the source is in `src/github.com/etsy/hound` ... which means I'm generally operating from the root of that directory.

`ui/bindata.go` definitely cannot be in the .gitignore. When we change the UI, we have to regenerate that file and check it in so that users who are installing with `go get` will have all the web assets.